examples/components/components-ngFor/app/index.ts

```diff
@@ -1,18 +0,0 @@
-import { NgModule } from '@angular/core';
-import { BrowserModule } from '@angular/platform-browser';
-import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import { App } from './app.component.ts';
-import { Counter } from './counter.component.ts';
-
-@NgModule({
-  imports: [
-    BrowserModule
-  ],
-  declarations: [
-    App
-  ],
-  bootstrap: [ App ]
-})
-export class AppModule {
-}
-platformBrowserDynamic().bootstrapModule(AppModule);
```

Comment:
When refactoring, this file was renamed/relocated as main.ts in the root folder.